activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -68,12 +68,12 @@ def singularize(word, locale = :en)
     #   camelize(underscore('SSLError'))        # => "SslError"
     def camelize(term, uppercase_first_letter = true)
       string = term.to_s
-      # String#camelize takes a symbol (:upper or :lower), so we match that to avoid surprises:
-      upcase = true == uppercase_first_letter || :upper == uppercase_first_letter
-      if upcase
-        string = string.sub(/^[a-z\d]*/) { |match| inflections.acronyms[match] || match.capitalize! || match }
-      else
+      # String#camelize takes a symbol (:upper or :lower), so here we also support :lower to keep the methods consistent.
+      should_downcase = !uppercase_first_letter || uppercase_first_letter == :lower
+      if should_downcase
```

Comment:
Just enough readable without assigning onetime local variable.

```suggestion
      if !uppercase_first_letter || uppercase_first_letter == :lower
```

Reply:
👍